salt/utils/pycrypto.py

```diff
@@ -41,10 +41,26 @@
 log = logging.getLogger(__name__)
 
 
-def secure_password(length=20, use_random=True):
+def secure_password(length=20, use_random=True, **kwargs):
```

Comment:
Please add the supported keyword arguments and their default values, we don't need this to be `**kwargs`